aggregator_api/src/lib.rs

```diff
@@ -113,13 +115,23 @@ async fn auth_check(conn: &mut Conn, State(cfg): State<Arc<Config>>) -> impl Han
         constant_time::verify_slices_are_equal(bearer_token.as_ref(), key.as_ref()).is_ok()
     }) {
         // Authorization succeeds.
+        conn.set_state(cfg);
         return None;
     }
 
     // Authorization fails.
     Some((Status::Unauthorized, Halt))
 }
 
+async fn get_config(_: &mut Conn, State(config): State<Arc<Config>>) -> Json<AggregatorApiConfig> {
+    Json(AggregatorApiConfig {
+        dap_url: config.public_dap_url.clone(),
```

Comment:
Remind me, I think I missed context on this choice: why do we need the aggregator to supply its own URL? The downstream client can't infer this based on how the aggregator is being reached?

Reply:
Correct; the aggregator api may be hosted at a different domain from the dap server or the same domain at a different path. There's no way for divviup-api to know how the dap server and aggregator-api are configured in relation to each other